precipitation_obs/create_artifact.jl

```diff
@@ -0,0 +1,24 @@
+using Downloads
+
+using ClimaArtifactsHelper
+
+const FILE_URL = "https://caltech.box.com/shared/static/k1or5d0d9xdvyfiytl1os7z341monkzn.nc"
```

Comment:
Shouldn't the FILE_URL point to something in https://data.caltech.edu/records/z24s9-nqc90?

Reply:
let me try!